examples/gnrc_tcp_cli/Makefile

```diff
@@ -31,7 +31,7 @@ CFLAGS += -DGNRC_TCP_MSS=1220
 CFLAGS += -DGNRC_TCP_MSL=2500000
 
 # Target Peer Address
-CFLAGS += -DTARGET_ADDR=\"fe80::68b3:a3ff:feb8:2d41\"
+CFLAGS += -DTARGET_ADDR=\"fe80::5c38:e9ff:fe76:6195\"
```

Comment:
These are Artifacts to Speedup Testing. I'll clean the makefiles Thursday morning ;)